packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -300,10 +325,20 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			this.filter = { ...defaultFilter, ...this.filter };
 		}
 
+		if (changes["fixedHeader"]) {
```

Comment:
Could you add some tests for this now that the property can be changed at any time?